prow/plugins/trigger/trigger.go

```diff
@@ -266,20 +266,10 @@ func TrustedUser(ghc trustedUserClient, onlyOrgMembers bool, trustedApps []strin
 		}
 	}
 
-	// Determine if there is a second org to check. If there is no secondary org or they are the same, the result
-	// is the same because the user already failed the check for the primary org.
-	if trustedOrg == "" || trustedOrg == org {
-		// the if/else is only to improve error messaging
-		if onlyOrgMembers {
-			return TrustedUserResponse{IsTrusted: false, Reason: notMember.String()}, nil // No trusted org and/or it is the same
-		}
-		return TrustedUserResponse{IsTrusted: false, Reason: (notMember | notCollaborator).String()}, nil // No trusted org and/or it is the same
-	}
-
 	// Check the second trusted org.
-	member, err := ghc.IsMember(trustedOrg, user)
+	member, err := ghc.IsMember(org, user)
 	if err != nil {
-		return errorResponse, fmt.Errorf("error in IsMember(%s): %w", trustedOrg, err)
+		return errorResponse, fmt.Errorf("error in IsMember(%s): %w", org, err)
```

Comment:
This does not update the function in the correct way. This change causes us to check the membership twice. The conditional on line 271 should always be true now that we are removing trustedOrg so we should always execute that code path rather than never executing it.